src/calcPrimeNumbers/App.java

```diff
@@ -0,0 +1,14 @@
+package calcPrimeNumbers;
+
+public class App {
+
+	public static void main(String[] args) {
+		int myint;
+		do {
+			PrimeNumbers app = new PrimeNumbers();
```

Comment:
Names should be self-explanatory. When you read PrimeNumbers what do you think this is? You named the variable app. If it's an app, maybe say say PrimeNumbersApp? What about the App class your in? Which one is the app? :P Naming can be hard sometimes :)) It should describe what the class purpose is.

For example I would rename the App class to PrimeNumberApp, because this is the app, here we have the main function.

And then, PrimeNumbers class I would rename it to PrimeNumbersService for example, because this is not the app, but a service that computes prime numbers inside the app.